main.py

```diff
@@ -260,49 +355,17 @@ def main(cl_arguments):
                           to_train, opt_params, schd_params,
                           args.shared_optimizer, args.load_model, phase="main")
 
-    # Select model checkpoint from main training run to load
+    # For checkpointing logic 
     if not args.do_target_task_training:
         log.info("In strict mode because do_target_task_training is off. "
                  "Will crash if any tasks are missing from the checkpoint.")
         strict = True
```

Comment:
@W4ngatang  why would we want strict to equal False if target_Task_Training = 0?

Reply:
Hmm, I didn't author this (I think Sam did?), but I would guess it's because there's not really a good reason to evaluate an untrained target task classifier. More often than not, it's probably a configuration error?

Reply:
That sounds right...

Reply:
In that case, we should have an assert for this (if it's a configuration error) rather than letting this pass by (even if there are a bunch of warning and exclamation points in the warning)

Reply:
Made a new issue